client/src/Component/Checkbox.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from "react";
+import Form from "react-bootstrap/Form";
+
+
+
+const Checkbox = (props) => {
+
+    return (
+        <Form>
+            <Form.Check type="checkbox" id="checkbox" aria-label="checkbox" defaultChecked= {props.isChecked} onChange={(event) => props.onChange(event.target.checked)} />
```

Comment:
- It doesn't make sense to have an id attribute with a static value here. If we have 5 of these checkboxes on the page, then we have 5 elements on the page with `id=checkbox`. id's should be unique on a page, so there should be a maximum of 1 `id=checkbox` on the page. Instead, we can remove this id attribute since we're not using it
- All form controls should have a label, and here we're giving it one with `aria-label`. However, `checkbox` doesn't describe what this checkbox does. The checkbox lets us select a user from the table, so a more meaningful label could be the name of the client. Let's pass the client name into this `Checkbox` component and use it for the value of `aria-label`